### PR TITLE
Add anchor element

### PR DESCRIPTION
**Pull request by houfio**

### CODE REVIEW

**Review by Wollaars — DISMISSED** [review]

File: server/features/manuals/Mutation.ts [hunk 1]
```diff
@@ -1,7 +1,118 @@
...
+          return null;
+        }
+
+        await db.$transaction([
+          db.manualStep.deleteMany({
+            where: {
+              manual: { id }
+            }
+          }),
+          ...steps.map((step) => db.manualStep.create({
```
Comment (IndyV) [inline server/features/manuals/Mutation.ts:79]: Thanks I hate it 🎅

Comment (houfio) [reply server/features/manuals/Mutation.ts:79]: right back at you

File: src/pages/_error.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import { Empty } from '../components/templates/Empty';
+
+type Props = {
+  statusCode?: number;
+};
+
+export default function Error({ statusCode }: Props) {
+  return (
+    <StyledMessage>
+      ¯\_({statusCode})_/¯
```
Comment (IndyV) [inline src/pages/_error.tsx:13]: Nice 🐙

**Review by IndyV — APPROVED** [review]
👌

**Pull request merged**